File: golang/random_number_guess.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func random(min int, max int) int {
	return rand.Intn(max-min) + min
}

func main() {
	rand.Seed(time.Now().UnixNano())
	number := random(0, 100)

	for {
		var user_input int
		fmt.Print("Enter your number: ")
		fmt.Scan(&user_input)

		if user_input > number {
			fmt.Println("Enter lower number")
		} else if user_input < number {
			fmt.Println("Enter bigger number")
		} else {
			fmt.Println("Congratulation!...")
			break
		}
	}
}

File: javascript/random_number_guess.js
const prompt = require('prompt-sync')({sigint: true});

const number = Math.floor(Math.random() * 101);

while (true) {
    const user_input = prompt("Enter number: ");

    if(+user_input > number) {
        console.log("Enter lower number");
    } else if(+user_input < number) {
        console.log("Enter bigger number");
    } else {
        console.log("Congratulation!...");
        break;
    }
}